compiler/tests-compiler/unix_fs.ml

```diff
@@ -172,11 +175,14 @@ let%expect_test "Unix.readlink_EINVAL" =
   compile_and_run
     {|
   (match Unix.readlink "." with
-  | exception Unix.Unix_error (Unix.EINVAL, syscall, path) -> print_endline ("EINVAL: " ^ syscall)
+  (* Linux & Mac raise Unix.EINVAL *)
+  | exception Unix.Unix_error (Unix.EINVAL, syscall, path) -> print_endline "EXPECTED ERROR"
+  (* Windows raises Unix.EUNKNOWNERR *)
+  | exception Unix.Unix_error (Unix.EUNKNOWNERR(code), syscall, path) -> print_endline "EXPECTED ERROR"
```

Comment:
We also have a `Unix.EUNKNOWNERR` now because Windows doesn't know how to handle readlink on a directory.